src/main/groovy/org/pih/warehouse/invoice/InvoiceAdjustmentStatus.groovy

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2012 Partners In Health.  All rights reserved.
+ * The use and distribution terms for this software are covered by the
+ * Eclipse Public License 1.0 (http://opensource.org/licenses/eclipse-1.0.php)
+ * which can be found in the file epl-v10.html at the root of this distribution.
+ * By using this software in any fashion, you are agreeing to be bound by
+ * the terms of this license.
+ * You must not remove this notice, or any other, from this software.
+ **/
+package org.pih.warehouse.invoice
+
+import org.pih.warehouse.core.StatusType
+
+enum InvoiceAdjustmentStatus {
```

Comment:
I am not sure if `InvoiceAdjustmentStatus` naming is a safe bet here. I think it would be better as a `OrderAdjustmentPaymentStatus` or `OrderAdjustmentInvoiceStatus` to match what it is displaying. `InvoiceAdjustment`  might be something completely different